subo/command/se2_deploy.go

```diff
@@ -23,7 +23,10 @@ import (
 )
 
 type deployData struct {
-	SE2Version       string
+	SE2Version string // TODO: remove me after Beta-6 release
+
+	SE2Tag           string
+	E2CoreTag        string
```

Comment:
Prefer `XTag` here because they represent tags, not versions (which are similar but different because of prefixed v).